packages/shared/src/svg/StarWarsLogoIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+
+export default function StarWarsLogoIcon(
```

Comment:
@kopancek I think u've confused branches

Reply:
you're right 😢